styles.css

```
@@ -13,19 +14,194 @@
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
-
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
 /* ------------------------------- 2. Typography ----------------------------------- */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6{
+    font-family: 'Merriweather' , sans-serif;
+
+}
+
+body{
+    font-family: 'Roboto' , sans-serif;
+    border: darkred solid 1px;
+
 
+
+}
+h2{
+    font-size: 18px;
+}
 /* ------------------------------- 3. Layout --------------------------------------- */
 
+
 /* --------------- 3.1 Reusables ----------------------- */
 
+
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.header {
+    background-image: linear-gradient(to bottom right, #4845E4, #5722CD) ;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    position: relative;
+    color: aliceblue;
+
+
+}
+.outer-container-header{
+    display: flex;
+    justify-content: center;
+    position: relative;
+    padding: 0 0 50px 0;
+    flex-basis: 1200px;
+    flex-wrap: wrap;
+}
+.navigation-bar{
+    display: flex;
+    justify-content: flex-end;
+    flex-basis: 1200px;
+    padding-top: 25px;
+    padding-bottom: 25px;
+}
```

Hier zou ik een aantal dingen anders doen, want je wilt eigenlijk niet een basis van 1200px hebben, want de pagina kan groter én kleiner zijn dan dat. Alleen je wilt wel dat het er mooi uit blijft zien
```suggestion
.outer-container-header{
    display: flex;
    justify-content: center;
    position: relative;
    padding: 0 0 50px 0;
    flex-wrap: wrap;
    flex-direction: column;
    align-items: center;
}
.navigation-bar{
    display: flex;
    justify-content: flex-end;
    width: 100%;
    padding-top: 25px;
    padding-bottom: 25px;
}
```